web/templates/mobile_survey/index.html.eex

```diff
@@ -4,6 +4,7 @@
     window.mixEnv = "<%= Mix.env %>"
     window.respondentId = <%= @respondent_id %>
     window.token = "<%= @token %>"
+    window.introMessage = "<%= raw @mobile_web_intro_message %>"
```

Comment:
Why `raw`?  Isn't this leading to some script injection?

Reply:
Good catch! Thanks! I've removed the raw() call and renamed the property name to clarify its content. It was the best solution I've found considering the class you give us about the subject (thanks a lot for that!) and a few script injection tests I did manually. If this solution isn't enough, let's iterate!